src/components/accommodation/form/fields/AccommodationFormAccessibility.jsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { FormGroup } from "reactstrap";
+import { Field } from "formik";
+import { useTranslation } from "react-i18next";
+import { AccommodationFormFields } from "components/accommodation/AccommodationFormFields";
+import FormCheckbox from "components/atoms/form/FormCheckbox";
+
+const checkboxField = ({ name, label }) => {
+    return (
+        <Field
+            id={name}
+            key={name}
+            name={name}
+            type="checkbox"
+            label={label}
+            component={FormCheckbox}
+            inline
+        />
+    );
+};
+
+const AccommodationFormPets = () => {
```

Comment:
I believe 'Pets' is not relevant anymore in this context, if we also include information about disabilities 😄